openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -549,6 +594,38 @@ def test_credit_course_counted_complete_for_verified(self, mock_completed_course
         mock_completed_course_runs.return_value = [{'course_run_id': course_run_key, 'type': 'verified'}]
         self.assertEqual(meter._is_course_complete(course), True)
 
+    def test_course_grade_results(self, mock_get_programs):
+        grade_percent = .8
+        with mock_passing_grade(percent=grade_percent):
```

Comment:
The test is probably fine, but I'm confused about how it works.  Probably no code changes needed, but where does "mock_passing_grade" come in and why are we mocking a "passing" grade, vs just the grade?

Reply:
This is to mock out the internals of the grading system and instead just have the grading system return the value we specify. I think it used to be only mocking out passing grades, but then it got generalized to any grade: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/tests/utils.py#L15